memphis/producer.py

```diff
@@ -213,7 +281,41 @@ async def produce(
                 )
             raise MemphisError(str(e)) from e
 
+    async def _multi_station_produce(
+        self,
+        message,
+        ack_wait_sec: int = 15,
+        headers: Union[Headers, None] = None,
+        async_produce: Union[bool, None] = None,
+        msg_id: Union[str, None] = None,
+        producer_partition_key: Union[str, None] = None,
+        producer_partition_number: Union[int, -1] = -1
+    ):
+        tasks = []
+        for sn in self.station_name:
+            tasks.append(
```

Comment:
instead of working with tasks, make it work consecutively and return an error to the user if something occurs